src/constants.ts

```diff
@@ -28,18 +28,18 @@ export const ACTIVATION_KEYS_PROD_URL =
 export const ACTIVATION_KEYS_STAGE_URL =
   'https://console.stage.redhat.com/insights/connector/activation-keys';
 export const COMPLIANCE_AND_VULN_SCANNING_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security_guide/chap-compliance_and_vulnerability_scanning';
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/6/html/security_guide/chap-compliance_and_vulnerability_scanning';
 export const CREATING_IMAGES_WITH_IB_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index';
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index';
 export const MANAGING_WITH_DNF_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/managing_software_with_the_dnf_tool/index';
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/managing_software_with_the_dnf_tool/index';
 export const CREATING_IMAGES_WITH_IB_SERVICE_URL =
-  'https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/creating_customized_rhel_images_using_the_image_builder_service';
+  'https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/creating_customized_rhel_images_using_the_image_builder_service';
```

Comment:
This link seems to be dead, @ochosi can I replace it with `https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/8/html/creating_customized_images_by_using_insights_image_builder/index` (CREATING_IMAGES_WITH_IB_URL) or is there a better link we could use?

![image](https://github.com/osbuild/image-builder-frontend/assets/49452678/79e3fe71-f588-4b76-871a-eead17b81b38)

Reply:
All is good, I haven't refreshed and missed this PR https://github.com/osbuild/image-builder-frontend/pull/2211 from @croissanne :sweat_smile: